spec/game_spec.rb

```diff
@@ -0,0 +1,76 @@
+require './lib/game.rb'
+require './lib/player.rb'
```

Comment:
- [ ] OPTIONAL - Consider using require_relative in your spec file as it's a Ruby convention.